src/astro/databases/base.py

```diff
@@ -24,6 +24,15 @@
 from astro.sql.table import Metadata, Table
 
 
+class DatabaseCustomError(ValueError, AttributeError):
```

Comment:
@sunank200 Any reason for inheriting from ValueError and AttributeError as opposed to Exception?

Reply:
These are the relevant errors which we want to catch in the location.